amplify/backend/api/colonycdapp/schema.graphql

```diff
@@ -8,492 +8,963 @@ input AMPLIFY {
+"""
 input CreateUniqueColonyInput {
+  """Unique identifier for the Colony. This is the Colony's contract address."""
   id: ID!
+  """Display name of the Colony."""
   name: String!
+  """Unique identifier for the Colony's native token."""
   colonyNativeTokenId: ID!
+  """Type of the Colony (regular or MetaColony)."""
   type: ColonyType
+  """Status information for the Colony."""
   status: ColonyStatusInput
+  """Metadata related to the Colony's creation on the blockchain."""
   chainMetadata: ChainMetadataInput
+  """Version of the currently deployed Colony contract."""
   version: Int!
 }
 
+"""
+Input data to use when creating or changing a user profile
+"""
 input ProfileInput {
-  id: ID # nullable since resolver will use User / Colony id by default
+  """The unique identifier for the user profile."""
+  id: ID
+  """The URL of the user's avatar image."""
   avatar: String
+  """The URL of the user's thumbnail image."""
   thumbnail: String
+  """The display name of the user."""
   displayName: String
+  """A short description or biography of the user."""
   bio: String
+  """The user's location (e.g., city or country)."""
   location: String
+  """The user's personal or professional website."""
   website: AWSURL
+  """The user's email address."""
   email: AWSEmail
+  """Any additional metadata or settings related to the user profile."""
   meta: ProfileMetadataInput
 }
 
+"""
+Input data for a user's reputation within a Domain in a Colony. If no `domainId` is passed, the Root Domain is used.
+A `rootHash` can be provided, to get reputation at a certain point in the past.
+"""
 input GetUserReputationInput {
+  """The Ethereum wallet address of the user."""
   walletAddress: String!
+  """The Ethereum address of the Colony."""
   colonyAddress: String!
+  """The ID of the Domain within the Colony. If not provided, defaults to the Root Domain."""
   domainId: Int
+  """The root hash of the reputation tree at a specific point in time."""
   rootHash: String
 }
 
+"""
+Input data for updating an extension's information within a Colony, based on the Colony ID and extension hash.
+The hash is generated like so: `keccak256(toUtf8Bytes(extensionName))`, where `extensionName` is the name of the extension contract file in the Colony Network
+"""
 input UpdateExtensionByColonyAndHashInput {
+  """The unique identifier for the Colony."""
   colonyId: ID!
+  """The hash of the extension to be updated."""
   hash: String!
+  """A flag to indicate whether the extension is deprecated."""
   isDeprecated: Boolean
+  """A flag to indicate whether the extension is deleted."""
   isDeleted: Boolean
+  """A flag to indicate whether the extension is initialized."""
   isInitialized: Boolean
+  """The version of the extension."""
   version: Int
+  """The Ethereum address of the user who installed the extension."""
   installedBy: String
+  """The timestamp when the extension was installed."""
   installedAt: AWSTimestamp
 }
 
+"""
+???: what is this?
```

Comment:
Stores current version that is a available in the network of Colony and extensions.
key can be COLONY or extension hash.